explainaboard/metrics/qa_table_text_hybrid.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
+
+import abc
+from dataclasses import dataclass
+import itertools
+from typing import Optional
+
+import numpy as np
+
+from explainaboard.metrics.auxiliary import qa_table_text_hybrid_auxiliary as eval_util
+from explainaboard.metrics.metric import (
+    Metric,
+    MetricConfig,
+    MetricStats,
+    SimpleMetricStats,
+)
+from explainaboard.metrics.registry import metric_config_registry
+from explainaboard.utils.preprocessor import ExtractiveQAPreprocessor, Preprocessor
+from explainaboard.utils.typing_utils import unwrap_or
+
+
+class HybridQAMetric(Metric):
```

Comment:
Rename to `QATatMetric`, following our renaming of the processor? Similarly below.